testpilot/frontend/static-src/app/main.js

```diff
@@ -57,6 +59,17 @@ app.extend({
       });
       this.webChannel.sendMessage(type, data);
     });
+  },
+
+  // Send to GA
+  sendToGA(type, data) {
+    if (typeof(ga) !== 'undefined') {
+      data.hitType = type;
+      if (data.outboundURL) {
+        data.hitCallback = () => document.location = data.outboundURL;
+      }
+      ga('send', data);
```

Comment:
I think the `if (typeof(ga) !== 'undefined') {` line at the start of `sendToGA` does the existence check, here